internal/app/machined/pkg/runtime/v1alpha1/bootloader/sdboot/sdboot.go

```diff
@@ -163,9 +165,17 @@ func Probe(disk string, options options.ProbeOptions) (*Config, error) {
 //nolint:gocyclo
 func (c *Config) KexecLoad(r runtime.Runtime, disk string) error {
 	_, err := ProbeWithCallback(disk, options.ProbeOptions{}, func(conf *Config) error {
+		efiCtx := efivario.NewDefaultContext()
+
+		// we read the next entry to kexec as set by installer
```

Comment:
also confused here on the logic. 

when systemd-boot works, which entry does it kexec to?

Default or Selected?

As I don't think `KexecLoad` should set the EFI variables, but rather installing the bootloader should set the entry?